frontend/common/src/components/form/Select/SelectFieldV2.tsx

```diff
@@ -189,7 +189,7 @@ const SelectFieldV2 = ({
     <div data-h2-margin="base(0, 0, x.125, 0)">
       <InputWrapper
         {...{ label, context, error }}
-        inputId={id}
+        inputId={name}
```

Comment:
This fixes 'Unsaved changes' text not displaying in stories but feels strange in terms of naming. I couldn't find a better solution though.

Reply:
Yeah, I was able to take a look and I don't think there is a better way 😢